pages/docs/reference/whatsnew12.md

```diff
@@ -0,0 +1,460 @@
+---
+type: doc
+layout: reference
+title: "What's New in Kotlin 1.2"
+---
+
+# What's New in Kotlin 1.2
+
+## Table of Contents
+
+* [Multiplatform projects](#multiplatform-projects-experimental)
+* [Other language features](#other-language-features)
+* [Standard library](#standard-library)
+* [JVM backend](#jvm-backend)
+* [JavaScript backend](#javascript-backend)
+
+# Docs page draft: What's New in Kotlin 1.2
+
+## Multiplatform Projects (experimental)
+
+Multiplatform projects are a new **experimental** feature in Kotlin 1.2, allowing you to reuse code between target platforms supported by Kotlin – JVM, JavaScript and (in the future) Native. In a multiplatform project, you have three kinds of modules:
+
+* A *common* module contains code that is not specific to any platform, as well as declarations without implementation of platform-dependent APIs.
+* A *platform* module contains implementations of platform-dependent declarations in the common module for a specific platform, as well as other platform-dependent code.
+* A regular module that targets a specific platform and can either be a dependency of platform modules or depend on platform modules.
+
+When you compile a multiplatform project for a specific platform, the code for both the common and platform-specific parts is generated.
+
+A key feature of the multiplatform project support is the possibility to express dependencies of common code on platform-specific parts through **expected and actual declarations**. An expected declaration specifies an API (class, interface, annotation, top-level declaration etc.). An actual declaration is either a platform-dependent implementation of the API or a typealias referring to an existing implementation of the API in an external library. Here's an example:
+
+In common code:
+
+```kotlin
+// expected platform-specific API:
+expect fun hello(world: String): String
+
+fun greet() {
+    // usage of the expected API:
+    val greeting = hello("multi-platform world")
+    println(greeting)
+}
+
+expect class URL(spec: String) {
+    open fun getHost(): String
+    open fun getPath(): String
+}
+```
+
+In JVM platform code:
+
+```kotlin
+actual fun hello(world: String): String =
+    "Hello, $world, on the JVM platform!"
+
+// using existing platform-specific implementation:
+actual typealias URL = java.net.URL
+```
+
+See the [documentation](http://kotlinlang.org/docs/reference/multiplatform.html) for details and steps to build a 
+multiplatform project.
+
+### Writing multiplatform unit tests
```

Comment:
I don't think it's needed to repeat that here.

Reply:
Removed the section.